package-lock.json

```diff
@@ -1,32 +1,32 @@
 {
```

Comment:
Are we sure that we need lock file in repo?

FYI @pvlt

Reply:
Yep, we need it. lockfile ensures that we all use the same dependencies and optimize dependencies installation in ci. [package-lock.json](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json)